helpers/cache.js

```diff
@@ -16,9 +16,14 @@ module.exports.connect = function (cacheEnabled, config, logger, cb) {
 		return cb(null, { cacheEnabled: cacheEnabled, client: null });
 	}
 
+	Object.keys(config).forEach(function (key){
```

Comment:
If we explicitly set the value of a key to null, it will be coerced to a string value. As stated here, https://github.com/NodeRedis/node_redis#sending-commands

I have tested this feature as well, and setting a parameter to null converts it into string and throws this error: 

> node_redis: Deprecated: The AUTH command contains a "null" argument.
> This is converted to a "null" string now and will return an error from v.3.0 on.
> Please handle this in your code to make sure everything works as you intended it to.